### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by letmejustputthishere**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,8 +50,8 @@ Use the Internet Identity canister in your local dfx project by adding the follo
   "canisters": {
     "internet_identity": {
       "type": "custom",
-      "candid": "https://github.com/dfinity/internet-identity/releases/download/release-2023-11-17/internet_identity.did",
-      "wasm": "https://github.com/dfinity/internet-identity/releases/download/release-2023-11-17/internet_identity_dev.wasm.gz",
+      "candid": "https://github.com/dfinity/internet-identity/releases/latest/download/internet_identity.did",
+      "wasm": "https://github.com/dfinity/internet-identity/releases/latest/download/internet_identity_dev.wasm.gz",
```
Comment (nmattia) [inline README.md:54]: This is not ideal as this doesn't pin the release. An alternative would be to update the README on new releases!

Comment (letmejustputthishere) [reply README.md:54]: why do we need to pin the release in the readme?

Comment (nmattia) [reply README.md:54]: people copy/paste it and then they get different versions depending on when they first ran their first `dfx` command; it's just best practice

Comment (nmattia) [reply README.md:54]: Added an automatic update: https://github.com/dfinity/internet-identity/pull/2226

**Comment by nmattia** [comment]
Closing because of recently added automation to update the README; feel free to reopen

**Pull request closed without merging**